apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1.SmokeTests/ImageAnnotatorSmokeTest.g.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+using Google.Api.Gax;
+using Google.Api.Gax.Grpc;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Vision.V1.SmokeTests
+{
+    public class ImageAnnotatorSmokeTest
+    {
+        public static int Main(string[] args)
+        {
+            // Create client
+            ImageAnnotatorClient client = ImageAnnotatorClient.Create();
+
+            // Initialize request argument(s)
+            IEnumerable<AnnotateImageRequest> requests = new[]
+            {
+                new AnnotateImageRequest
+                {
+                    Image = new Image
+                            {
```

Comment:
I don't like this formatting. At some point I'll see if I can persuade the codegen to do this better.

Reply:
Ooh, yes - I hadn't noticed that. At least it's a smoke test rather than a snippet...